SPARCore/GADArticle.h

```diff
@@ -1,22 +1,26 @@
 #import <Foundation/Foundation.h>
+#import "GADRemoteModel.h"
 
-@interface GADArticle : NSObject
+@interface GADArticle : GADRemoteModel
 
+@property (nonatomic, strong) NSString *datePublished;
+@property (nonatomic, strong) NSString *brief;
+@property (nonatomic, strong) NSString *headerImage;
```

Comment:
I think an NSURL property might be more apt. Currently we're getting an NSString from the request which I assume is just string form of the url we're wanting to use, so we can easily just convert the NSString to an NSURL.